docs/en_US/sqls/query_language_elements.md

```diff
@@ -13,7 +13,12 @@ Kuiper provides a variety of elements for building queries. They are summarized
 | [ORDER BY](#ORDER BY) | Order the rows by values of one or more columns.             |
 | [HAVING](#HAVING)     | HAVING specifies a search condition for a group or an aggregate. HAVING can be used only with the SELECT expression.             |
 |                       |                                                              |
+### Use reserved words
+These elements and other predefined operators, functions are reserved words. To use the reserved words as the column name and the table name etc., they need to be quoted by backtick.
```

Comment:
If the field name includes Chinese, or blank character, is it supported? If we support that, we should write the doc as well.